.github/workflows/pr-check.yaml

```diff
@@ -23,55 +23,184 @@ permissions:
+    runs-on: ubuntu-24.04
+    timeout-minutes: 60
+    env:
+      SKIP_INSTALLATION: true
+    steps:
+      - uses: actions/checkout@v5
+        with: 
+          path: minc-extension
+
+      # Checkout podman desktop
+      - uses: actions/checkout@v5
+        with:
+          repository: podman-desktop/podman-desktop
+          ref: main
+          path: podman-desktop
+
+      - name: Download prerelease binaries
+        run: |
+          # Fetch the latest tag from the testing-prereleases repository
+          LATEST_TAG=$(curl -s https://api.github.com/repos/podman-desktop/testing-prereleases/tags | jq -r '.[0].name')
+          LATEST_VERSION=${LATEST_TAG:1} #removing the leading 'v' character
+          echo "Latest prerelease tag: $LATEST_TAG"
+          echo "Latest prerelease version: $LATEST_VERSION"
+          
+          # Construct the download URL for the tarball
+          DOWNLOAD_URL="https://github.com/podman-desktop/testing-prereleases/releases/download/${LATEST_TAG}/podman-desktop-${LATEST_VERSION}.tar.gz"
+          echo "Downloading from: $DOWNLOAD_URL"
+          
+          # Download the tarball
+          curl -L -o podman-desktop-prerelease.tar.gz "$DOWNLOAD_URL"
+          
+          # Extract tarball (it already contains a folder)
+          tar -xzf podman-desktop-prerelease.tar.gz
+          
+          # Rename the extracted folder to podman-desktop-prerelease
+          mv podman-desktop-${LATEST_VERSION} podman-desktop-prerelease
+          
+          # Make the podman-desktop binary executable
+          chmod +x podman-desktop-prerelease/podman-desktop
+          
+          # Set environment variable for subsequent steps
+          PODMAN_DESKTOP_BINARY="${GITHUB_WORKSPACE}/podman-desktop-prerelease/podman-desktop"
+          echo "PODMAN_DESKTOP_BINARY=${PODMAN_DESKTOP_BINARY}" >> $GITHUB_ENV
+          echo "Podman Desktop binary path: ${PODMAN_DESKTOP_BINARY}"
+          
+          # Clean up tarball
+          rm podman-desktop-prerelease.tar.gz
+          
+          echo "Extracted to: podman-desktop-prerelease"
+          ls -la podman-desktop-prerelease
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          run_install: false
+          package_json_file: ./podman-desktop/package.json
+
+      - uses: actions/setup-node@v6
+        with:
+          node-version: 22
+      
+        # on windows, wsl kernel update before this
+      - name: Update Podman
+        uses: redhat-actions/podman-install@main
+
+      - name: Revert unprivileged user namespace restrictions in Ubuntu 24.04
+        run: |
+          # allow unprivileged user namespace
+          sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
+
+      - name: Ensure getting current HEAD version of the test framework
+        working-directory: ./minc-extension
+        run: |
+          # workaround for https://github.com/containers/podman-desktop-extension-bootc/issues/712
```

Comment:
you can do only this:
```
pnpm add -D @podman-desktop/tests-playwright@next
```
and that is it. not need to further update package.json in this PR since it is done "inline" in the wf run